doc/vault-integration.md

```diff
@@ -43,29 +43,33 @@ Commands below assumes the VM is an Ubuntu instance.
 
 #### Install Vault
 Run the following commands in your VM.
+
+Install vault on Ubuntu following the vault [docs](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install)
+
+Configure vault
 ```bash
-# install vault and enable the service
-sudo apt update && sudo apt install vault
 sudo systemctl enable vault.service
 
 # create a configuration file for vault. NOTE: this creates a vault service with TLS disabled. 
 # This is done to make the configuration step easy to follow only. TLS should be enabled for real workloads.
-cat <<< "ui = true
-
+cat <<< 'ui = true
+disable_mlock = true
```

Comment:
Why do we need this?

Reply:
It was genereted when from the default config. I will remove it
>disable_mlock (bool: false) – Disables the server from executing the mlock syscall. mlock prevents memory from being swapped to disk. Disabling mlock is not recommended unless using integrated storage. Follow the additional security precautions outlined below when disabling mlock. This can also be provided via the environment variable VAULT_DISABLE_MLOCK.